application/src/main/java/org/opentripplanner/graph_builder/module/osm/OsmElevatorKey.java

```diff
@@ -0,0 +1,3 @@
+package org.opentripplanner.graph_builder.module.osm;
+
+public record OsmElevatorKey(long nodeId, OsmEntityType osmEntityType, long entityId) {}
```

Comment:
This one can be package-private I believe.

Reply:
changed to the suggestion